File: URI/1 - INICIANTE/Python/1160 - CrescimentoPopulacional.py
z = int(input())
for y in range(0,z):
    pa, pb, g1, g2 = input().split()
    pa = int(pa)
    pb = int(pb)
    g1 = float(g1)
    g2 = float(g2)
    t  = 0
    while pa <= pb:
        pa += int(pa*(g1/100.0))
        pb += int(pb*(g2/100.0))
        t += 1
        if(t > 100):
            t = 101
            break
    if t == 101:
        print('Mais de um seculo.')
    else:
        print('{} anos.'.format(t))

File: URI/1 - INICIANTE/C/1160 - CrescimentoPopulacional.c
#include <stdio.h>

int main()
{
	int x, tempo, i = 0;
	long pa, pb;
	double g1, g2;
	
	scanf(" %d", &x);
	
	while(i < x){
		scanf(" %ld %ld %lf %lf", &pa, &pb, &g1, &g2);
		tempo = 0;
		while(pa <= pb)
		{
			pa += (long)((pa*g1)/100);
			pb += (long)((pb*g2)/100);
			tempo += 1;
			if(tempo > 100)
			{
				tempo = 101;
				break;
			}
		}
		if(tempo == 101){
			printf("mais de um seculo.\n");
		}else{
			printf("%d anos.\n", tempo);
		}
		i+=1;
	}
	
	return 0;
}
